vault/logical_system_helpers.go

```diff
@@ -32,7 +32,7 @@ var (
 	}
 
 	sysInitialize = func(b *SystemBackend) func(context.Context, *logical.InitializationRequest) error {
-		return nil
+		return ceSysInitialize(b)
```

Comment:
nit: the `ceSysInitialize` function matches the signature of this lambda, and all the lambda does is call it, so I think you can simplify this whole call to:

```go
sysInitialize = ceSysInitialize
```

Or am I missing something?